bigtop-packages/src/rpm/ranger/SPECS/ranger.spec

```diff
@@ -31,7 +31,7 @@
 %define usr_lib_sqoop %{parent_dir}/usr/lib/sqoop
```

Comment:
Could you remove Ranger related changes from this PR? It should be done in separate JIRA/PR.